### PR TITLE
402, 405 and partly 404 – requested icons

### PR DESCRIPTION
**Pull request by z4-xz**
Icons requested by @warpedwood in #402, and some of @TobiasDev #404 #405

### CODE REVIEW

**Comment by Donnnno** [comment]
Great work!
A small suggestion: Try to use as less unnecessary shapes as possible, like how it's explained here:

> When you bring an icon to Frost try to make it minimal. The easiest and most noticeable way to do so is to set an icon free of its background. Twitter is a good example! There are two parts: a bird and a circle surrounding it. The bird is the center of the icon and the part associated with Twitter while the circle is not unique to the platform and can be removed.

**Comment by z4-xz** [comment]
Thanks, I'll try to be more focused on simplicity, just as you said

File: app/src/main/res/xml/appfilter.xml [hunk 1]
```diff
@@ -1795,6 +1804,8 @@
     <!-- Microsoft Excel -->
     <item component="ComponentInfo{com.microsoft.office.excel/com.microsoft.office.apphost.LaunchActivity}" drawable="excel" />
 
+    <!-- Microsoft Launcher -->
+    <item component="ComponentInfo{com.microsoft.launcher/com.microsoft.launcher.Launcher}" drawable="microsoftlauncher"/>
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:1808]:
```suggestion
    <item component="ComponentInfo{com.microsoft.launcher/com.microsoft.launcher.Launcher}" drawable="microsoftlauncher"/>

```

Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:1808]: Could you possibly add a newline after this entry?

**Review by dkanada — COMMENTED** [review]
Looks like there's a merge conflict with FlorisBoard, are you familiar with rebasing or merge commits? I can fix it otherwise.

File: app/src/main/res/xml/appfilter.xml (hunk 1 above)
Comment (z4-xz) [reply app/src/main/res/xml/appfilter.xml:1808]: Both problems fixed

**Comment by z4-xz** [comment]
@dkanada Is it ok?

File: app/src/main/res/xml/appfilter.xml [hunk 2]
```diff
@@ -1042,6 +1048,9 @@
     <!-- Floatplane -->
     <item component="ComponentInfo{com.floatplane.floatplane/com.floatplane.floatplane.MainActivity}" drawable="floatplane" />
 
+    <!-- FlorisBoard -->
+    <item component="ComponentInfo{dev.patrickgold.florisboard/dev.patrickgold.florisboard.SettingsLauncherAlias}" drawable="florisboard"/>
+
```
Comment (dkanada) [inline app/src/main/res/xml/appfilter.xml:1053]:
```suggestion
```

**Pull request merged**